migtests/tests/oracle-tests/ora-trunc-tests/ora-date-trunc-test/validate

```diff
@@ -17,9 +17,8 @@ def migration_completed_checks(tgt):
 		print(f"table_name: {table_name}, row_count: {got_row_count[table_name]}")
 		assert row_count == got_row_count[table_name]
 	
-	view_list = tgt.get_objects_of_type("VIEW", "test_mysql_views")
-	print("view_list:", view_list)
-	assert len(view_list) == 1
```

Comment:
Why is this removed?

Reply:
The test schema was changed keeping in mind the objective and hence the assertion too.